docs/patterns/health-endpoint-monitoring.md

```diff
@@ -2,8 +2,8 @@
 title: Health Endpoint Monitoring pattern
 titleSuffix: Cloud Design Patterns
 description: Implement functional checks in an application that external tools can access through exposed endpoints at regular intervals.
-author: dragon119
-ms.date: 06/23/2017
+author: magrande
+ms.date: 09/28/2021
 ms.topic: conceptual
 ms.service: architecture-center
 ms.subservice: design-pattern
```

Comment:
Can you please add the

`categories` and `products` metadata tags with related items in each array.

You can look at other content to see what that looks like if you don't know.  This is a build error that we'll want addressed.